### PR TITLE
feat: Release react-progress as unstable

### LINKED ISSUE #25273: chore: Release react-progress to unstable
This issue is to track work to release `Progress` to unstable.

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR releases `react-progress` to unstable. Fixes #25273

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                |       Size (minified/GZIP) |
| ------------------------------------------------------------------------------------------------ | -------------------------: |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/Progress.fixture.js'>Progress</abbr> | `11.66 kB`<br />`4.619 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/82650ef8b3d436c039e33f3eb8a658a3d878064f'>82650ef8b3d436c039e33f3eb8a658a3d878064f</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/Avatar-mount_base.html">1245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/Avatar-mount.html">1248</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/Button-mount_base.html">940</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/Button-mount.html">878</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProvider-mount_base.html">1497</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProvider-mount.html">1514</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProviderWithTheme-mount_base.html">636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProviderWithTheme-mount.html">624</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">586</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">529</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">628</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">617</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/MakeStyles-mount_base.html">1914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/MakeStyles-mount.html">1920</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/SpinButton-mount_base.html">2258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25226/perf-test-react-components/SpinButton-mount.html">2507</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: change/@fluentui-react-components-5b50946f-c569-450d-87b2-4276e9c0b31a.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "add react-progress to react-components",
```
Comment (layershifter) [inline change/@fluentui-react-components-5b50946f-c569-450d-87b2-4276e9c0b31a.json:3]:
```suggestion
  "comment": "feat: add react-progress to react-components",
```

**Comment by layershifter** [comment]
Please add a notice about the component being unstable to docs (like other components have):

image

File: change/@fluentui-react-progress-bfc00986-7094-47cb-a1ab-2a3f3b7a892c.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "add react-progress to react-components",
```
Comment (Hotell) [inline change/@fluentui-react-progress-bfc00986-7094-47cb-a1ab-2a3f3b7a892c.json:3]:
```suggestion
  "comment": "feat: add react-progress to react-components",
```

Comment (Hotell) [reply change/@fluentui-react-progress-bfc00986-7094-47cb-a1ab-2a3f3b7a892c.json:3]: https://github.com/microsoft/fluentui/issues/25270

File: change/@fluentui-react-components-5b50946f-c569-450d-87b2-4276e9c0b31a.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  "packageName": "@fluentui/react-components",
+  "email": "ololubek@microsoft.com",
+  "dependentChangeType": "patch"
```
[the remark this answers is not in the document]
Comment (tomi-msft) [reply change/@fluentui-react-components-5b50946f-c569-450d-87b2-4276e9c0b31a.json:6]: great question! the last component that was added to unstable was `Persona` and the type was `patch` as well, so I was just following that convention: #25008

File: change/@fluentui-react-components-5b50946f-c569-450d-87b2-4276e9c0b31a.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "feat: add react-progress to react-components",
```
Comment (khmakoto) [inline change/@fluentui-react-components-5b50946f-c569-450d-87b2-4276e9c0b31a.json:3]:
```suggestion
  "comment": "feat: add react-progress to react-components/unstable",
```

File: change/@fluentui-react-progress-bfc00986-7094-47cb-a1ab-2a3f3b7a892c.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "feat: add react-progress to react-components",
```
Comment (khmakoto) [inline change/@fluentui-react-progress-bfc00986-7094-47cb-a1ab-2a3f3b7a892c.json:3]:
```suggestion
  "comment": "feat: add react-progress to react-components/unstable",
```

**Review by khmakoto — APPROVED** [review]

**Pull request merged**